dev/line_chart.html

```diff
@@ -44,6 +45,10 @@
 //   d.values = d.values.slice((i*0.5)*100, (i+1)*100)
 // })
 
+data.series.forEach(d => {
+  d.values = d.values.map(dB => ({...dB, value: !!~~(Math.random() * 8) ? dB.value : null}))
```

Comment:
Right. I use this file solely for developing, and this line is just to tweak the data to add nul for this specific development. But I think it would be nice to make it more clear and also to use some of these experiments as examples in the examples folder

Reply:
I ported it to the data manager and added a nullRatio option. And it's not using this pattern anymore.